Makefile

```diff
@@ -0,0 +1,50 @@
+# Time-stamp: < Makefile (2017-01-10 10:03) >
+BUILD=build
+NAME=docker-tunnel
+TAG=bubak4
```

Comment:
well, in fact I did not expect you will integrate my Makefile at all, so I was not extra careful -- it is a bit Linux specific as this is my platform of choice -- I just included the Makefile just for my own reference as I use Makefile a lot to automate docker container builds.  I will make a bit nicer, though it always involves a bit of work to modify it specific needs.

Feel free to ignore the Makefile.